v2_jobs.go

```diff
@@ -79,14 +84,20 @@ func (s *JobStatus) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-type Task struct {
-	ID       string `json:"uuid"`
-	ExitCode uint8  `json:"exit_code"`
-}
+func (s *JobsServiceOp) List(ctx context.Context,
+	ID string,
+	r *JobsListRequest) ([]Job, *Response, error) {
```

Comment:
This is now a cosmetic change and I must say if we include it it will be the **only** multi-line method declaration in the entire project. Can you revert the change?

```suggestion
func (s *JobsServiceOp) List(ctx context.Context, ID string, r *JobsListRequest) ([]Job, *Response, error) {
```